components/mpas-seaice/src/shared/mpas_seaice_forcing.F

```diff
@@ -3339,12 +3339,6 @@ subroutine get_data_iceberg_fluxes(domain)
     real(kind=RKIND) :: &
          scaling
 
-    ! dc including as parameters here so as not to create new namelist options
-    real(kind=RKIND), parameter :: &
-         specificHeatFreshIce = 2106.0_RKIND, & ! specific heat of fresh ice J * kg^-1 * K^-1
```

Comment:
I'd suggest we keep these, but as namelist option for iceberg temperature, and specific heat of fresh ice used as a constant.